MailCore/Cache/ContactManager/ContactManager+Merge.swift

```diff
@@ -155,7 +159,8 @@ extension ContactManager {
         let cleanupRealm = getRealm()
         try? cleanupRealm.safeWrite {
             for emailToDelete in toDelete {
-                guard let objectToDelete = cleanupRealm.object(ofType: MergedContact.self, forPrimaryKey: emailToDelete) else {
+                guard let objectToDelete = cleanupRealm.object(ofType: MergedContact.self, forPrimaryKey: emailToDelete),
+                      !objectToDelete.isInvalidated else {
```

Comment:
I don't think `isInvalidated` is needed because we get the object directly from realm

Reply:
Good point, let me check in the documentation and change it